src/components/Apps/AccountListTable/const.js

```diff
@@ -160,6 +163,9 @@ export const accountOtherActions = (vm) => [
     name: 'MoveToOther',
     title: vm.$t('MoveToAsset'),
     type: 'primary',
+    has: () => {
+      return !vm.asset
+    },
     callback: ({ row }) => {
       vm.accountCreateUpdateTitle = vm.$t('MoveToOther')
       vm.$route.query.flag = 'move'
```

Comment:
I'm sorry, but I need more context about the codes you're referring to. Could you please provide me with the code samples so I can assist you better?